rosdep/base.yaml

```diff
@@ -4774,6 +4781,7 @@ libssl-dev:
   gentoo: [dev-libs/openssl]
   nixos: [openssl]
   openembedded: [openssl@openembedded-core]
+  opensuse: [libopenssl-devel]
```

Comment:
How does this get tanslaged through https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.15927/standard ?

I think I see libopenssl1_1-devel